tests/Microsoft.Bot.Builder.TestBot.Json/TestOptionsMiddleware.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.TestBot.Json
+{
+    public class TestOptionsMiddleware : IMiddleware
```

Comment:
>TestOptionsMiddleware [](start = 17, length = 21)

This should be part of Microsoft.Bot.Builder.Dialogs.Adaptive.Testing to be used by everyone.